src/Tools/PublicApi/mark-shipped.ps1

```diff
@@ -0,0 +1,46 @@
+[CmdletBinding(PositionalBinding=$false)]
+param ()
+
+Set-StrictMode -version 2.0
+$ErrorActionPreference = "Stop"
+
+function MarkShipped([string]$dir) {
+    $shippedFilePath = Join-Path $dir "PublicAPI.Shipped.txt"
+    $shipped = Get-Content $shippedFilePath
+    $unshippedFilePath = Join-Path $dir "PublicAPI.Unshipped.txt"
+    $unshipped = Get-Content $unshippedFilePath
+    $removed = @()
+    $removedPrefix = "*REMOVED*";
+    Write-Host "Processing $dir"
+
+    foreach ($item in $unshipped) {
+        if ($item.StartsWith($removedPrefix)) {
+            $item = $item.Substring($removedPrefix.Length)
+            $removed += $item
+        }
+        else {
+            $shipped += $item
+        }
+    }
+
+    $shipped | ?{ -not $removed.Contains($_) } |  Out-File $shippedFilePath -Encoding Ascii
```

Comment:
Ideally yes.  Practically it causes merge hell because our code fixer doesn't sort.